src/main/features/response/form/models/impactOfDispute.ts

```diff
@@ -0,0 +1,22 @@
+import { Serializable } from 'models/serializable'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+import { MaxLength } from 'forms/validation/validators/maxLengthValidator'
+
+export class ImpactOfDispute implements Serializable<ImpactOfDispute> {
+
+  @MaxLength(ValidationConstraints.FREE_TEXT_MAX_LENGTH, { message: DefaultValidationErrors.FREE_TEXT_TOO_LONG })
```

Comment:
i would just use `ValidationErrors` here instead of Default, any reason it's renamed the class here?